static/map.js

```diff
@@ -42,6 +42,7 @@ function pad(num, size) {
 document.getElementById('pokemon-checkbox').checked = getFromStorage("displayPokemons", "true");
 document.getElementById('gyms-checkbox').checked = getFromStorage("displayGyms", "true");
 document.getElementById('coverage-checkbox').checked = getFromStorage("displayCoverage", "true");
+document.getElementById('pokestops-checkbox').checked = getFromStorage("displayPokestops", true);
```

Comment:
change `true` to `"true"`, it doesn't get recognized so the pokestops are not displayed by default.